Improvement and documentation of packages data (#152)
openfisca/openfisca.org: CONTRIBUTING.md

description:  # in both languages, 200 characters max
  en: Explore legislation formulas and parameters modelled with OpenFisca.
  fr: Explorez les formules et les paramètres de la législation modélisée.  # deepl.com can be used for automated translation

> package: OpenFisca-Country-Template  # name of the OpenFisca package used by this reuse

Reviewer: ```suggestion
package: OpenFisca-Country-Template  # name of the Python package that this reuse depends on
```

It is strongly recommended to use an image compression tool such as [ImageOptim](https://imageoptim.com) to reduce the image weight. Choosing “lossy” settings at 90% quality will usually yield over 50% compression gains.

## Referencing your package

> By referencing your package on the [dedicated page](https://openfisca.org/en/packages/) of the OpenFisca website you give it visibility so that others can use it. We do our best to make this process as easy as possible, [contact us]((mailto:contact@openfisca.org?subject=Referencing%20package)) if you need.

Reviewer: ```suggestion
By referencing your package on the [dedicated page](https://openfisca.org/en/packages/) of the OpenFisca website you give it visibility so that others can use it. We do our best to make this process as easy as possible, [contact us](mailto:contact@openfisca.org?subject=Referencing%20package) if you need.
```
